src/components/VHeader/VHeader.vue

```diff
@@ -1,44 +1,30 @@
 <template>
   <header
-    class="flex py-4 px-4 md:px-7 items-stretch z-40 w-full bg-white md:gap-x-2 gap-y-4"
+    class="flex px-4 md:px-7 items-stretch z-40 w-full bg-white gap-x-2 gap-y-4 flex-wrap md:flex-nowrap"
     :class="{
-      'flex-wrap': isSearchRoute,
+      'py-3 md:py-4': !isHeaderScrolled,
+      'py-3': isHeaderScrolled,
```

Comment:
`py-3` as being applied either way, so we can move it to the static `class` list.

```suggestion
      'md:py-4': !isHeaderScrolled,
```

Reply:
That was actually an incorrect implementation, thank you for catching it! It should be py-4 if the header isn't scrolled, and py-3 if it is, no matter the screen size.